README.md

```diff
@@ -73,6 +73,20 @@ puck --extra-files config/*.yml
 
 The more advanced `:extra_files` feature, where the destination paths within the JAR file is specified as described below, is not available as a command line option.
 
+If your application is depending on another Jar you can merge it into the application Jar using the `:merge_archives` option:
```

Comment:
I think this description needs more context. It's rare that you need this feature, it's not just because you depend on another Jar, but that you have special requirements.

Perhaps: _In some circumstances you need to merge another Jar or zip file with the Jar created by Puck. This can be useful if have Java libraries, like JDBC drivers, that need to be available to the top level class loader. This is possible using the `:merge_archives` option:_